config/routes.rb

```diff
@@ -3,14 +3,14 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root to: 'static_pages#home'
 
-  match 'i/signup/(*path)' => redirect { |_,request| "signup/#{request.params[:path]}?#{request.params.except('path').to_query}" }, via: :all
+  get 'i/signup/(*path)' => redirect { |_,request| "signup/#{request.params[:path]}?#{request.params.except('path').to_query}" }, via: :get
```

Comment:
get via get seems redundant. I thought you only needed a via clause if you were using match.

Reply:
I missed this but yeah, I think `via` only does anything if you use `match`.

Reply:
@Dantemss and you said to only use these for GET requests, right? So just remove `via:`?